setup_logging.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+function setup_logging() {
+    echo "installing Cluster Logging and EFK operators..."
+    oc create -f cluster-logging/eo-namespace.yaml
+    oc create -f cluster-logging/clo-namespace.yaml
+    oc create -f cluster-logging/eo-og.yaml
+    oc create -f cluster-logging/eo-sub.yaml
+    oc create -f cluster-logging/clo-og.yaml
+    oc create -f cluster-logging/clo-sub.yaml
+    # wait until the CRD is available before creating the CR
+    NEXT_WAIT_TIME=0
+    while [[ -z `oc get crd clusterloggings.logging.openshift.io 2>/dev/null` ]]; do
+		if [[ ${NEXT_WAIT_TIME} -eq 300 ]]; then \
+		   OPERATOR_GROUP_NAME=`oc get operatorgroups --output=name -n openshift-logging | grep "openshift-logging-"`
+		   SUBSCRIPTION_NAME=`oc get subscription --output=name -n openshift-logging | grep "cluster-logging"`
+		   echo "reached timeout of waiting for CRD clusterloggings.logging.openshift.io to be available in the cluster - see following info for debugging:"
+		   echo "================================ Subscription =================================="
+		   oc get ${SUBSCRIPTION_NAME} -n openshift-logging -o yaml
+		   exit 1
+		fi
+		echo "$(( NEXT_WAIT_TIME++ )). attempt of waiting for CRD clusterloggings.logging.openshift.io to be available in the cluster..."
```

Comment:
maybe the sentence is better as `attempt number [NEX_WAIT_TIME++] of waiting for CRD .... `

the way the sentence is read may be a bit off if it's `two attempt of wating for CRD ...`

Reply:
> the way the sentence is read may be a bit off if it's two attempt of wating for CRD ...

there won't be no `two` in the sentence - the sentences that will be printed out will look like:
```
0. attempt of waiting for CRD clusterloggings.logging.openshift.io to be available in the cluster...
1. attempt of waiting for CRD clusterloggings.logging.openshift.io to be available in the cluster...
2. attempt of waiting for CRD clusterloggings.logging.openshift.io to be available in the cluster...
3. attempt of waiting for CRD clusterloggings.logging.openshift.io to be available in the cluster...
4. attempt of waiting for CRD clusterloggings.logging.openshift.io to be available in the cluster...
5. attempt of waiting for CRD clusterloggings.logging.openshift.io to be available in the cluster...
```
which is - from my point of view - totally fine

Reply:
I've update the output in 83d7649 and it now prints the following logs:
```
Installing the Cluster Logging and EFK operators...
namespace/openshift-operators-redhat created
namespace/openshift-logging created
operatorgroup.operators.coreos.com/openshift-operators-redhat created
subscription.operators.coreos.com/elasticsearch-bk2kf created
operatorgroup.operators.coreos.com/openshift-logging-xgxwp created
subscription.operators.coreos.com/cluster-logging created
attempt 1 of waiting for CRD clusterloggings.logging.openshift.io to be available in the cluster...
attempt 2 of waiting for CRD clusterloggings.logging.openshift.io to be available in the cluster...
attempt 3 of waiting for CRD clusterloggings.logging.openshift.io to be available in the cluster...
attempt 4 of waiting for CRD clusterloggings.logging.openshift.io to be available in the cluster...
attempt 5 of waiting for CRD clusterloggings.logging.openshift.io to be available in the cluster...
clusterlogging.logging.openshift.io/instance created
================================================================================================================
run 'oc get routes --namespace=openshift-logging' to get the URL of the Kibana dashboard once the pod is running
================================================================================================================
```